Could not read from file: D:\vite-project\node_modules\stream-browserify\web

### System Info

```
[ERROR] Could not read from file: D:\vite-project\node_modules\stream-browserify\web

    node_modules/@xenova/transformers/src/utils/hub.js:10:19:
      10 │ import stream from 'stream/web';
D:\vite-project\node_modules\esbuild\lib\main.js:1651
  let error = new Error(text);
              ^

Error: Build failed with 1 error:
node_modules/@xenova/transformers/src/utils/hub.js:10:19: ERROR: Could not read from file: D:\vite-project\node_modules\stream-browserify\web
    at failureErrorWithLog (D:\vite-project\node_modules\esbuild\lib\main.js:1651:15)
    at D:\vite-project\node_modules\esbuild\lib\main.js:1059:25
    at D:\vite-project\node_modules\esbuild\lib\main.js:1527:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  errors: [Getter/Setter],
  warnings: [Getter/Setter]
}
```
os:win11
node version:v21.6
npm config:
```
_auth = (protected)
access = null
all = false
allow-same-version = false
also = null
audit = true
audit-level = null
auth-type = "web"
before = null
bin-links = true
browser = null
ca = null
cache = "C:\\Users\\Administrator\\AppData\\Local\\npm-cache"
cache-max = null
cache-min = 0
cafile = null
call = ""
cert = null
cidr = null
color = true
commit-hooks = true
cpu = null
depth = null
description = true
dev = false
diff = []
diff-dst-prefix = "b/"
diff-ignore-all-space = false
diff-name-only = false
diff-no-prefix = false
diff-src-prefix = "a/"
diff-text = false
diff-unified = 3
dry-run = false
editor = "C:\\Windows\\notepad.exe"
engine-strict = false
fetch-retries = 2
fetch-retry-factor = 10
fetch-retry-maxtimeout = 60000
fetch-retry-mintimeout = 10000
fetch-timeout = 300000
force = false
foreground-scripts = false
format-package-lock = true
fund = true
git = "git"
git-tag-version = true
global = false
global-style = false
globalconfig = "D:\\nodejs\\etc\\npmrc"
heading = "npm"
; https-proxy = null ; overridden by user
if-present = false
ignore-scripts = false
include = []
include-staged = false
include-workspace-root = false
init-author-email = ""
init-author-name = ""
init-author-url = ""
init-license = "ISC"
init-module = "C:\\Users\\Administrator\\.npm-init.js"
init-version = "1.0.0"
init.author.email = ""
init.author.name = ""
init.author.url = ""
init.license = "ISC"
init.module = "C:\\Users\\Administrator\\.npm-init.js"
init.version = "1.0.0"
install-links = false
install-strategy = "hoisted"
json = false
key = null
legacy-bundling = false
legacy-peer-deps = false
link = false
local-address = null
location = "user"
lockfile-version = null
loglevel = "notice"
logs-dir = null
logs-max = 10
; long = false ; overridden by cli
maxsockets = 15
message = "%s"
node-options = null
noproxy = [""]
npm-version = "10.2.4"
offline = false
omit = []
omit-lockfile-registry-resolved = false
only = null
optional = null
os = null
otp = null
pack-destination = "."
package = []
package-lock = true
package-lock-only = false
parseable = false
prefer-dedupe = false
prefer-offline = false
prefer-online = false
prefix = "D:\\nodejs"
preid = ""
production = null
progress = true
provenance = false
provenance-file = null
; proxy = null ; overridden by user
read-only = false
rebuild-bundle = true
; registry = "https://registry.npmjs.org/" ; overridden by user
replace-registry-host = "npmjs"
save = true
save-bundle = false
save-dev = false
save-exact = false
save-optional = false
save-peer = false
save-prefix = "^"
save-prod = false
sbom-format = null
sbom-type = "library"
scope = ""
script-shell = null
searchexclude = ""
searchlimit = 20
searchopts = ""
searchstaleness = 900
shell = "C:\\Windows\\system32\\cmd.exe"
shrinkwrap = true
sign-git-commit = false
sign-git-tag = false
strict-peer-deps = false
; strict-ssl = true ; overridden by user
tag = "latest"
tag-version-prefix = "v"
timing = false
umask = 0
unicode = true
update-notifier = true
usage = false
user-agent = "npm/{npm-version} node/{node-version} {platform} {arch} workspaces/{workspaces} {ci}"
userconfig = "C:\\Users\\Administrator\\.npmrc"
version = false
versions = false
viewer = "browser"
which = null
workspace = []
workspaces = null
workspaces-update = true
yes = null

; "user" config from C:\Users\Administrator\.npmrc

https-proxy = null
proxy = null
registry = "https://registry.npmmirror.com"
strict-ssl = false

; "cli" config from command line options

long = true
```

### Environment/Platform

- [X] Website/web-app
- [ ] Browser extension
- [ ] Server-side (e.g., Node.js, Deno, Bun)
- [ ] Desktop app (e.g., Electron)
- [ ] Other (e.g., VSCode extension)

### Description

When i import the pipeline module,the above error accured
`import { pipeline } from "@xenova/transformers";`

### Reproduction

When i import the pipeline module,the above error accured
`import { pipeline } from "@xenova/transformers";`

--- a/package.json
+++ b/package.json
@@ -74,8 +74,7 @@
     "path": false,
     "url": false,
     "sharp": false,
-    "onnxruntime-node": false,
-    "stream/web": false
+    "onnxruntime-node": false
   },
   "publishConfig": {
     "access": "public"

--- a/src/utils/hub.js
+++ b/src/utils/hub.js
@@ -7,15 +7,9 @@
 
 import fs from 'fs';
 import path from 'path';
-import stream from 'stream/web';
 
 import { env } from '../env.js';
 import { dispatchCallback } from './core.js';
-
-if (!globalThis.ReadableStream) {
-    // @ts-ignore
-    globalThis.ReadableStream = stream.ReadableStream; // ReadableStream is not a global with Node 16
-}
 
 /**
  * @typedef {Object} PretrainedOptions Options for loading a pretrained model.     

